delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjectionFields.scala

```diff
@@ -90,6 +90,7 @@ object CompositeViewProjectionFields {
   final case class ElasticSearchProjectionFields(
       id: Option[Iri] = None,
       query: SparqlConstructQuery,
+      indexGroup: Option[String],
```

Comment:
any reason for the name difference? `indexPrefix` vs `indexGroup`?

Reply:
It is the same, it meant to name them indexGroup as there is already a prefix. I will fix that